.github/workflows/gopy.yml

```diff
@@ -28,16 +28,40 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: "3.11"
+          python-version: "3.11.10"
 
       - name: Install pybindgen
         run: |
           python3 -m pip install pybindgen
           go install golang.org/x/tools/cmd/goimports@latest
           go install github.com/go-python/gopy@latest
 
-      - name: Build
+      - name: Build python module
         run: |
           cd src/pkg/client
-          gopy pkg -author="Photoroom" -email="team@photoroom.com" -url="" -name="datago" -version="0.99" .
-          ls datago/*
+          gopy pkg -author="Photoroom" -email="team@photoroom.com" -name="datago" .
+          export DESTINATION="../../../build"
+          mkdir -p $DESTINATION/datago
+          mv datago/* $DESTINATION/datago/.
+          mv setup.py $DESTINATION/.
+          mv Makefile $DESTINATION/.
+          mv README.md $DESTINATION/.
+          rm LICENSE MANIFEST.in
+          cd ../../../build
+
+      - name: Install python module
+        run: |
+          ls
+          cd build
+          python3 -m pip install --user -v -e .
+
+      - name: Run the python unit tests
+        env:
```

Comment:
run tests in the python scope, check that everything runs end to end. Said tests are almost empty at the moment, but they do show that we can pull data within the interpreter scope